weaver/wps_restapi/swagger_definitions.py

```diff
@@ -2788,17 +2814,20 @@ class PostProcessJobsEndpoint(ProcessPath):
 
 class GetJobsQueries(ExtendedMappingSchema):
     detail = ExtendedSchemaNode(Boolean(), description="Provide job details instead of IDs.",
-                                default=False, example=True, missing=drop)
+                                default=False, example=True, missing=False)
     groups = ExtendedSchemaNode(String(),
                                 description="Comma-separated list of grouping fields with which to list jobs.",
-                                default=False, example="process,service", missing=drop)
-    page = ExtendedSchemaNode(Integer(), missing=drop, default=0, validator=Range(min=0))
-    limit = ExtendedSchemaNode(Integer(), missing=drop, default=10)
-    status = JobStatusEnum(missing=drop)
+                                default=False, example="process,service", missing=False)
+    page = ExtendedSchemaNode(Integer(), missing=0, default=0, validator=Range(min=0))
+    limit = ExtendedSchemaNode(Integer(), missing=10, default=10, validator=Range(min=0, max=10000))
+    datetime = DateTimeInterval(missing=None, default=None)
+    status = JobStatusEnum(missing=None, default=None)
     process = AnyIdentifier(missing=None)
-    provider = ExtendedSchemaNode(String(), missing=drop, default=None)
+    service = ExtendedSchemaNode(String(), missing=None, default=None)
     sort = JobSortEnum(missing=drop)
-    tags = ExtendedSchemaNode(String(), missing=drop, default=None,
+    access = JobAccess(missing=None, default=None)
```

Comment:
The items that don't need an explicit int default (ie: that use `None`) should probably keep `missing=drop`. 
When `deserialize` is called, if the parameter was not provided in the request, it will be removed from the filters. 
Because `find_jobs` provides the defaults, it is valid to not provide the filters.